codegen/swift/swift-entities/src/main/resources/client/SwiftAPIClientCore.mustache

```diff
@@ -117,6 +117,12 @@ public protocol Deserializable {
     static func deserialize(data: NSData?) -> Result<Self, NSError>
 }
 
+extension Void : Deserializable {
```

Comment:
@evgenbakumenko I'm a bit worried about this part: to my understanding such file can appear in multiple places in the codebase. Will it cause any issues or compilation errors - I do not know if it lives in its own namespace or scope?